src/components/standings.rs

```diff
@@ -15,6 +21,7 @@ pub struct StandingsState {
     pub standings: Vec<Division>,
```

Comment:
I think adding something like `league_standings: Vec<Standing>` here would simplify this. when the data is updated in the `update` method, you could just generate this with something like `self.league_standings = self.get_teams_by_record();`